proto/dydxprotocol/indexer/events/events.proto

```diff
@@ -238,22 +238,16 @@ message SubaccountUpdateEventV1 {
 // placement or triggering of a conditional order, or the removal of a
 // stateful order.
 message StatefulOrderEventV1 {
-  reserved 2, 3;
+  reserved 1, 2, 3;
 
   // The type of event that this StatefulOrderEvent contains.
   oneof event {
-    StatefulOrderPlacementV1 order_place = 1;
```

Comment:
Can you just note that it's deprecated in docs with the deprecated tag?
Don't remove any of the validation / checking.
I think we'd want to maintain compatability with older versions of the protocol which can still send this property.

Reply:
Makes sense, changed!